onedal/svm/backend/svm_py.cpp

```diff
@@ -320,7 +312,7 @@ void svm_infer<Task>::infer(PyObject *data,
                      .set_biases(biases_table);
     if constexpr (std::is_same_v<Task, svm::task::classification> ||
                   std::is_same_v<Task, svm::task::nu_classification>) {
-        model.set_first_class_label(0).set_second_class_label(1);
+        model.set_first_class_response(0).set_second_class_response(1);

```

Comment:
But old methods with `_label` naming will work too? (until the next major release?)

Reply:
yes, they were marked as deprecated (see https://github.com/oneapi-src/oneDAL/pull/1755/checks?check_run_id=2942887627)